src/widgets/SearchBar.js

```diff
@@ -18,7 +18,10 @@ import { debounce } from '../utilities/index';
  * with a magnifying glass icon and clear button.
  *
  * Debounces input by the user, such that the onChangeText callback
- * is only invoked once every `debounceTimeout`
+ * is only invoked once every `debounceTimeout`.
+ *
+ * NOTE: This component is exported MEMOIZED. See below for propsAreEqual
```

Comment:
Thinking back to when I was new to hooks, might be more helpful here to explain the why rather than the what, e.g.:

```
This component only re-renders when the |value| prop is updated.
```

Reply:
It's only here to try to minimize a mistake someone in the future might have wondering why their component might not be rendering as per normal. The comment you suggest is on the `propsAreEqual` function itself

Reply:
Sure, but it seems like you could just do the one comment? You could prob get away with it just in the docstring as you'd think `propsAreEqual` is fairly self-documenting in this instance.